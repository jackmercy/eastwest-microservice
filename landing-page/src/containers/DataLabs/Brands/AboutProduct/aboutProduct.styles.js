import { createUseStyles } from 'react-jss';

import { hexToRgbA, rem } from '../../../../utils/jss';
import COLORS from '../../../../constants/COLORS';
import BREAKPOINTS from '../../../../constants/BREAKPOINTS';

export default createUseStyles({
  primary: { color: COLORS.primary },
  root: { backgroundColor: hexToRgbA(COLORS.bgBlack01, 0.02) },
  content: {
    textAlign: 'center',
    paddingTop: rem(80),
    paddingBottom: rem(120),
  },
  title: {
    fontWeight: 900,
    fontSize: rem(72),
    lineHeight: rem(88),
    margin: 0,
    transition: 'font-size .5s ease',
  },
  description: {
    fontWeight: 600,
    fontSize: rem(24),
    lineHeight: rem(40),
    maxWidth: rem(1000),
    margin: [rem(48), 'auto', 0],
    transition: 'font-size .5s ease',
  },
  group: {
    marginTop: rem(80),
    display: 'flex',
    justifyContent: 'space-between',
  },
  groupItem: {
    margin: [rem(80), 'auto', 0],
    width: '100%',
    maxWidth: rem(208),
  },
  groupItemImage: { display: 'inline-block', maxHeight: rem(128) },
  groupItemValue: {
    fontWeight: 900,
    fontSize: rem(48),
    lineHeight: rem(64),
    marginTop: rem(32),
    marginBottom: 0,
  },
  groupItemLabel: {
    fontSize: rem(24),
    lineHeight: rem(24),
    marginTop: rem(16),
    marginBottom: 0,
  },

  [`@media only screen and (max-width: ${BREAKPOINTS.maxSm}px)`]: {
    content: { paddingTop: rem(40), paddingBottom: rem(40) },
    title: { fontSize: rem(32), lineHeight: rem(40) },
    description: { fontSize: rem(16), lineHeight: rem(24), marginTop: rem(16) },
    group: { marginTop: 0, display: 'block' },
  },
});
