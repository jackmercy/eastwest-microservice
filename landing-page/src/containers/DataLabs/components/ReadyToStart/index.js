import dynamic from 'next/dynamic';

export default dynamic(() => import('./ReadyToStart'), { ssr: false });
