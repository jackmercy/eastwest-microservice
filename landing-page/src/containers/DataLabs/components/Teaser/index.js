import dynamic from 'next/dynamic';

export default dynamic(() => import('./Teaser'), { ssr: false });
