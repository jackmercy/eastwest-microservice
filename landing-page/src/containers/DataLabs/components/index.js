export { default as Teaser } from './Teaser';
export { default as JoinUs } from './JoinUs';
export { default as Become } from './Become';
export { default as HowItWorks } from './HowItWorks';
export { default as ReadyToStart } from './ReadyToStart';
