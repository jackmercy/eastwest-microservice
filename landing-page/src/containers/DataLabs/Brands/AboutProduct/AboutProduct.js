import { memo, useMemo, useState } from 'react';
import { useIntl } from 'react-intl';
import CountUp from 'react-countup';
import VisibilitySensor from 'react-visibility-sensor';

import { Content } from '../../../components';

import useStyles from './aboutProduct.styles';

const AboutUs = () => {
  const intl = useIntl();
  const classes = useStyles();
  const companyInfo = useMemo(
    () => [
      {
        key: 'launch',
        value: 2021,
        countUpConfigs: { prefix: '', suffix: '' },
        image: '/static/images/data-labs/brands/about-01.png',
        label: intl.formatMessage({ id: 'DATA_LABS.ABOUT.01.LABEL' }),
        dataTest: 'about-product-launch',
      },
      {
        key: 'influencers',
        value: 45000,
        countUpConfigs: { prefix: '', suffix: '' },
        image: '/static/images/data-labs/brands/about-02.png',
        label: intl.formatMessage({ id: 'DATA_LABS.ABOUT.02.LABEL' }),
        dataTest: 'about-product-influencers',
      },
      {
        key: 'brands',
        value: 100,
        countUpConfigs: { prefix: '', suffix: '+' },
        image: '/static/images/data-labs/brands/about-03.png',
        label: intl.formatMessage({ id: 'DATA_LABS.ABOUT.03.LABEL' }),
        dataTest: 'about-product-brands',
      },
      {
        key: 'campaigns',
        value: 1000,
        countUpConfigs: { prefix: '', suffix: '+' },
        image: '/static/images/data-labs/brands/about-04.png',
        label: intl.formatMessage({ id: 'DATA_LABS.ABOUT.04.LABEL' }),
        dataTest: 'about-product-campaigns',
      },
    ],
    [intl],
  );
  const [isVisible, setVisibility] = useState(false);

  const handleVisibilityChange = (status) => {
    if (status) {
      setVisibility(status);
    }
  };

  return (
    <div className={classes.root}>
      <Content className={classes.content}>
        <h2 className={classes.title} data-test="about-product-title">
          {intl.formatMessage(
            { id: 'DATA_LABS.ABOUT.TITLE' },
            {
              span: (...chunks) => (
                <span className={classes.primary}>{chunks.join('')}</span>
              ),
            },
          )}
        </h2>
        <p
          className={classes.description}
          data-test="about-product-description"
        >
          {intl.formatMessage({ id: 'DATA_LABS.ABOUT.DESCRIPTION' })}
        </p>
        <div className={classes.group}>
          {companyInfo.map((item) => (
            <div
              key={item.key}
              className={classes.groupItem}
              data-test={item.dataTest}
            >
              <img
                alt={item.label}
                src={item.image}
                loading="lazy"
                className={classes.groupItemImage}
              />
              <h2 className={classes.groupItemValue}>
                <CountUp
                  end={item.value}
                  start={isVisible ? 0 : null}
                  {...item.countUpConfigs}
                >
                  {({ countUpRef }) => (
                    <VisibilitySensor
                      onChange={handleVisibilityChange}
                      delayedCall
                    >
                      <span ref={countUpRef} />
                    </VisibilitySensor>
                  )}
                </CountUp>
              </h2>
              <p className={classes.groupItemLabel}>{item.label}</p>
            </div>
          ))}
        </div>
      </Content>
    </div>
  );
};

export default memo(AboutUs);
