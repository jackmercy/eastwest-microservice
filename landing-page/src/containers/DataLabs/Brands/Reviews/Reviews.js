import { memo, useMemo } from 'react';
import { useIntl } from 'react-intl';

import { Content } from '../../../components';

import CustomCarousel from './CustomCarousel';
import useStyles from './reviews.styles';

const Reviews = () => {
  const intl = useIntl();
  const classes = useStyles();
  // TODO: remove images after integrate with APIs
  const listItems = useMemo(
    () => [
      // {
      //   name: 'Starprintvn',
      //   review:
      //     eslint-disable-next-line max-len
      //     'Our business relies entirly on the feedback of consumers. We print and develop packaging for product in every day use. Understanding exactly what the end consumers needs are, allows us to better develop solutions for our clients and literally gives us the edge over our competitors.',
      //   avatar: '/static/images/data-labs/brands/quote-01.svg',
      //   highlight: false,
      // },
      {
        name: 'RNW',
        review:
          // eslint-disable-next-line max-len
          'We build solutions for our real estate plaform on a continuous basis. Implmenting Data Labs for our User-Centered Design process allowed us to gather insights from our users to fine tune our products. Less costs on quantitative research and a brilliant form of Lead Generation.',
        avatar: '/static/images/data-labs/brands/quote-02.svg',
        highlight: true,
      },
      {
        name: 'KimOo Beatz',
        review:
          // eslint-disable-next-line max-len
          'Sampling for our music is subjective most of the time. We try to set or follow trends and engage with our audience on music released. Data Labs, allowed us to integrate data driven decision making on what direction we can take with our music to reach a better audience.',
        avatar: '/static/images/data-labs/brands/quote-03.svg',
        highlight: true,
      },
      {
        name: 'Toiya',
        review:
          // eslint-disable-next-line max-len
          'Creating content and gathering market insights became such an easy thing to do with Data Labs. We heavily rely on the feedback of our user base and manage requests on a continuous basis. Setting up multiple data streams has allowed us to allocate resources better and scale easier.',
        avatar: '/static/images/data-labs/brands/quote-04.svg',
        highlight: true,
      },
      {
        name: 'Voice Pick',
        review:
          // eslint-disable-next-line max-len
          'We work continuosly with data, generate reports and make recommendations to clients. Data is usually outdated, highly generalized and not specific. With Data Labs we get feedback from our users now. Itâ€™s accurate, reliable and gives us a selling point that competitors dont have.',
        avatar: '/static/images/data-labs/brands/quote-05.svg',
        highlight: true,
      },
    ],
    [],
  );

  return (
    <Content className={classes.content}>
      <h2 className={classes.title} data-test="reviews-title">
        {intl.formatMessage({ id: 'DATA_LABS.BRAND_QUOTES.TITLE' })}
      </h2>
      <CustomCarousel data={listItems} />
      <div className={classes.listItems} data-test="reviews-list-items">
        {listItems.filter(i => i.highlight).map((item, index) => (
          <div
            // eslint-disable-next-line react/no-array-index-key
            key={`review-${index}`}
            data-test="reviews-item"
            className={classes.itemLayout}
          >
            <div className={classes.itemWrapper}>
              <img
                alt={item.name}
                src="/static/images/data-labs/quotes.png"
                loading="lazy"
                className={classes.itemQuotesImage}
              />
              <p className={classes.itemReview}>{item.review}</p>
              <div className={classes.itemGroup}>
                <div className={classes.itemAvatarImageWrapper}>
                  <img
                    alt={item.name}
                    src={item.avatar}
                    loading="lazy"
                    className={classes.itemAvatarImage}
                  />
                </div>
                <p className={classes.itemName}>{item.name}</p>
              </div>
            </div>
          </div>
        ))}
      </div>
    </Content>
  );
};

export default memo(Reviews);
