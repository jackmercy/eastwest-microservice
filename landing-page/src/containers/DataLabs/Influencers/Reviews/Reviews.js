import { memo, useMemo } from 'react';
import { useIntl } from 'react-intl';

import { Content } from '../../../components';

import CustomCarousel from './CustomCarousel';
import useStyles from './reviews.styles';

const Reviews = () => {
  const intl = useIntl();
  const classes = useStyles();
  // TODO: remove images after integrate with APIs
  const listItems = useMemo(
    () => [
      {
        name: 'Duy Tran',
        review:
          // eslint-disable-next-line max-len
          'I have a very deep passion for fashion. I love wearing it and I enjoy watching buying it. Online shopping has become a norm. However, the use of excessive amounts of plastic is bothering me. It is amazing to see how many companies started to become more aware of hazardous packaging. Think straws in drinks and double wrapped bottle caps.',
        avatar: '/static/images/data-labs/influencers/quote-01.png',
        projects: 228,
        points: 1970,
        highlight: false,
      },
      {
        name: 'Thao Nguyen',
        review:
          // eslint-disable-next-line max-len
          'Having a voice with brands, I was able to contribute a lot in improving the product quality, without increasing the product costs. I am hoping other consumers can enjoy products more with this feedback.',
        avatar: '/static/images/data-labs/influencers/quote-02.png',
        projects: 210,
        points: 1512,
        highlight: true,
      },
      {
        name: 'Tuan Anh',
        review:
          // eslint-disable-next-line max-len
          'A great way to participate in small projects and get rewards. It doesn\'t take a lot time. I can participate during lunch breaks or in my spare time, without stressing my work load. It is a lot of fun and a great way of earning extra cash every month.',
        avatar: '/static/images/data-labs/influencers/quote-03.png',
        projects: 490,
        points: 1470,
        highlight: true,
      },
      {
        name: 'Nhi Dang',
        review:
          // eslint-disable-next-line max-len
          'Simple interface and easy to use, I receive participation requests in my inbox and can easily participate. The best part is, that we don‚Äôt get rewarded with vouchers or discounts but with currency. They told me crypto solutions are coming soon üòç',
        avatar: '/static/images/data-labs/influencers/quote-04.png',
        projects: 87,
        points: 470,
        highlight: true,
      },
      {
        name: 'Tai Nguyen',
        review:
          // eslint-disable-next-line max-len
          'I like to travel a lot. Data Labs, was able to pay for my last trip to the beach. I also got compensated for referring my friends to to become influencers. We actually worked on a few assignments together. Pretty amazing',
        avatar: '/static/images/data-labs/influencers/quote-05.png',
        projects: 130,
        points: 390,
        highlight: true,
      },
    ],
    [],
  );

  return (
    <Content className={classes.content}>
      <h2 className={classes.title} data-test="reviews-title">
        {intl.formatMessage({ id: 'DATA_LABS.BRAND_QUOTES.TITLE' })}
      </h2>
      <CustomCarousel data={listItems} />
      <div className={classes.listItems} data-test="reviews-list-items">
        {listItems
          .filter((i) => i.highlight)
          .map((item, index) => (
            <div
              // eslint-disable-next-line react/no-array-index-key
              key={`review-${index}`}
              data-test="reviews-item"
              className={classes.itemLayout}
            >
              <div className={classes.itemWrapper}>
                <img
                  alt={item.name}
                  src="/static/images/data-labs/quotes.png"
                  loading="lazy"
                  className={classes.itemQuotesImage}
                />
                <p className={classes.itemReview}>{item.review}</p>
                <div className={classes.itemGroup}>
                  <div className={classes.itemAvatarImageWrapper}>
                    <img
                      alt={item.name}
                      src={item.avatar}
                      loading="lazy"
                      className={classes.itemAvatarImage}
                    />
                  </div>
                  <div>
                    <p className={classes.itemName}>{item.name}</p>
                    <p className={classes.itemReward}>
                      <i className="icon-comment" />
                      {item.projects}
                    </p>
                    <p className={classes.itemReward}>
                      <i className="icon-circle-dollar" />
                      {item.points}
                    </p>
                  </div>
                </div>
              </div>
            </div>
          ))}
      </div>
    </Content>
  );
};

export default memo(Reviews);
