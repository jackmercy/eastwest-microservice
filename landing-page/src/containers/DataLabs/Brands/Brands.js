import { memo, useMemo } from 'react';
import { useIntl } from 'react-intl';

import {
  Teaser,
  JoinUs,
  Become,
  HowItWorks,
  ReadyToStart,
} from '../components';

import AboutProduct from './AboutProduct';
import Reviews from './Reviews';

const Brands = () => {
  const intl = useIntl();
  const sectionProps = useMemo(
    () => ({
      teaser: {
        image: '/static/images/data-labs/brands/teaser.png',
        mobileImage: '/static/images/data-labs/brands/teaser-mobile.png',
        purpose: intl.formatMessage({ id: 'DATA_LABS.TEASER.FOR_BRANDS' }),
        description: intl.formatMessage({
          id: 'DATA_LABS.TEASER.FOR_BRANDS.DESCRIPTION',
        }),
        otherText: intl.formatMessage({
          id: 'DATA_LABS.TEASER.FOR_INFLUENCERS',
        }),
        defaultReason: 4,
        isBrands: true,
      },
      howItWorks: {
        title: intl.formatMessage({
          id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.TITLE',
        }),
        description: intl.formatMessage({
          id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.DESCRIPTION',
        }),
        data: [
          {
            key: 'active',
            image: '/static/images/data-labs/brands/hiw-01.png',
            icon: '/static/images/data-labs/brands/hiw-icon-01.png',
            title: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.01.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.01.DESCRIPTION',
            }),
          },
          {
            key: 'tools',
            image: '/static/images/data-labs/brands/hiw-02.png',
            icon: '/static/images/data-labs/brands/hiw-icon-02.png',
            title: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.02.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.02.DESCRIPTION',
            }),
          },
          {
            key: 'segmentation',
            image: '/static/images/data-labs/brands/hiw-03.png',
            icon: '/static/images/data-labs/brands/hiw-icon-03.png',
            title: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.03.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.03.DESCRIPTION',
            }),
          },
          {
            key: 'matching-engine',
            image: '/static/images/data-labs/brands/hiw-04.png',
            icon: '/static/images/data-labs/brands/hiw-icon-04.png',
            title: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.04.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.04.DESCRIPTION',
            }),
          },
          {
            key: 'data',
            image: '/static/images/data-labs/brands/hiw-05.png',
            icon: '/static/images/data-labs/brands/hiw-icon-05.png',
            title: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.05.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BRANDS.HOW_IT_WORKS.05.DESCRIPTION',
            }),
          },
        ],
      },
      become: {
        title: intl.formatMessage({ id: 'DATA_LABS.BECOME_BRAND.TITLE' }),
        data: [
          {
            key: 'brand-become-01',
            title: intl.formatMessage({
              id: 'DATA_LABS.BECOME_BRAND.01.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BECOME_BRAND.01.DESCRIPTION',
            }),
            number: 165,
            image: '/static/images/data-labs/become-01.png',
            countUpConfigs: { prefix: '', suffix: '+' },
          },
          {
            key: 'brand-become-02',
            title: intl.formatMessage({
              id: 'DATA_LABS.BECOME_BRAND.02.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BECOME_BRAND.02.DESCRIPTION',
            }),
            number: 300,
            image: '/static/images/data-labs/become-02.png',
            countUpConfigs: { prefix: '', suffix: '+' },
          },
          {
            key: 'brand-become-03',
            title: intl.formatMessage({
              id: 'DATA_LABS.BECOME_BRAND.03.TITLE',
            }),
            description: intl.formatMessage({
              id: 'DATA_LABS.BECOME_BRAND.03.DESCRIPTION',
            }),
            number: 80,
            image: '/static/images/data-labs/become-03.png',
            countUpConfigs: { prefix: '', suffix: '+' },
          },
        ],
      },
      readyToStart: { defaultReason: 4 },
      joinUs: {
        description: intl.formatMessage({ id: 'DATA_LABS.JOIN_US.BRAND' }),
        defaultReason: 4,
      },
    }),
    [intl],
  );

  return (
    <>
      <Teaser {...sectionProps.teaser} />
      <HowItWorks {...sectionProps.howItWorks} />
      <AboutProduct />
      <Become {...sectionProps.become} />
      <ReadyToStart {...sectionProps.readyToStart} />
      <Reviews />
      <JoinUs {...sectionProps.joinUs} />
    </>
  );
};

export default memo(Brands);
