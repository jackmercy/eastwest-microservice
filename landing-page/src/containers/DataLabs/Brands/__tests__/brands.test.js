import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import preloadAll from 'jest-next-dynamic';

import { JoinUs, Teaser, Become, HowItWorks } from '../../components';
import Brands from '../Brands';
import AboutProduct from '../AboutProduct';
import Reviews from '../Reviews';

describe('Testing DataLabs > Brands', () => {
  let wrapper;

  beforeAll(async () => {
    // Teaser included in Brands
    // And they are using dynamic import, see Teaser.js
    // So we need wait for it to load all before we can assert anything
    await preloadAll();
    wrapper = mount(<Brands />);
  });

  it('should render match snapshot', () => {
    const wrapper = shallow(<Brands />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should have Teaser section', () => {
    expect(wrapper.find(Teaser)).toHaveLength(1);
  });

  it('should have How it works section', () => {
    expect(wrapper.find(HowItWorks)).toHaveLength(1);
  });

  it('should have About Product section', () => {
    expect(wrapper.find(AboutProduct)).toHaveLength(1);
  });

  it('should have Join Us section', () => {
    expect(wrapper.find(JoinUs)).toHaveLength(1);
  });

  it('should have Become Brands section', () => {
    expect(wrapper.find(Become)).toHaveLength(1);
  });

  it('should have Reviews section', () => {
    expect(wrapper.find(Reviews)).toHaveLength(1);
  });
});
