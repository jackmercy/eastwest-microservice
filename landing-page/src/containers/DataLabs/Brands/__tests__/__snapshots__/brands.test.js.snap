// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DataLabs > Brands should render match snapshot 1`] = `
<Fragment>
  <ForwardRef(LoadableComponent)
    defaultReason={4}
    description="Build products and promote your brand using data-driven user insights. Gather real data from real users, at any time. A product that connects brands with influencers by contributing feedback and information on developing products and  leveraging value towards your positioning in the market. Real data on your competitors, and consumer insights from your clients and users."
    image="/static/images/data-labs/brands/teaser.png"
    isBrands={true}
    mobileImage="/static/images/data-labs/brands/teaser-mobile.png"
    otherText="For Influencers"
    purpose="For Brands"
  />
  <Memo(HowItWorks)
    data={
      Array [
        Object {
          "description": "Register now for our closed alpha and gain advantage in your market right now! More than 45.000 influencers are available on our platform to participate in leveraging value for your brand.",
          "icon": "/static/images/data-labs/brands/hiw-icon-01.png",
          "image": "/static/images/data-labs/brands/hiw-01.png",
          "key": "active",
          "title": "Activate Data Labs",
        },
        Object {
          "description": "Get access to a data-driven product environment that allows you to create surveys, visualize data, gather insights, create focus groups, assign team members and properly setup your segmentation strategy.",
          "icon": "/static/images/data-labs/brands/hiw-icon-02.png",
          "image": "/static/images/data-labs/brands/hiw-02.png",
          "key": "tools",
          "title": "Choose one of our tools",
        },
        Object {
          "description": "You are able to determine a budget for each campaign and target a specific niche within your segment using diversified tools. Segmentation based on geographics, demographics, psychographics & behavioral aspects.",
          "icon": "/static/images/data-labs/brands/hiw-icon-03.png",
          "image": "/static/images/data-labs/brands/hiw-03.png",
          "key": "segmentation",
          "title": "Create audience segmentation",
        },
        Object {
          "description": "Our Matching engine will automatically select users from the platform based on your user segmentation. Once notified, they will review the project and decide if they want to participate.",
          "icon": "/static/images/data-labs/brands/hiw-icon-04.png",
          "image": "/static/images/data-labs/brands/hiw-04.png",
          "key": "matching-engine",
          "title": "Matching engine",
        },
        Object {
          "description": "Run multiple campaigns simultaneously to get better consumer insights regarding your brand and product. Ad-hoc or ahead of time. Multiple campaigns for multiple products.",
          "icon": "/static/images/data-labs/brands/hiw-icon-05.png",
          "image": "/static/images/data-labs/brands/hiw-05.png",
          "key": "data",
          "title": "Continuous streams of real data from real users",
        },
      ]
    }
    description="Setup continuous data streams, gather user insights, approach your audience directly and have them promote your brand as influencers. A great way to leverage value!"
    title="How it works"
  />
  <Memo(AboutUs) />
  <Memo(Become)
    data={
      Array [
        Object {
          "countUpConfigs": Object {
            "prefix": "",
            "suffix": "+",
          },
          "description": "Data-driven organizations are 165% more like to generate higher revenue growth and profitability than their peers.",
          "image": "/static/images/data-labs/become-01.png",
          "key": "brand-become-01",
          "number": 165,
          "title": "Data-driven Decision Making",
        },
        Object {
          "countUpConfigs": Object {
            "prefix": "",
            "suffix": "+",
          },
          "description": "The right strategy and architecture help manage data and analytics, and mitigate cost increases up to 300% in a user-centered environment.",
          "image": "/static/images/data-labs/become-02.png",
          "key": "brand-become-02",
          "number": 300,
          "title": "User-Centered feedback from real users",
        },
        Object {
          "countUpConfigs": Object {
            "prefix": "",
            "suffix": "+",
          },
          "description": "Build the data foundation for intelligent workflows, as 80% of the effort is know where you will have the highest chance of succes.",
          "image": "/static/images/data-labs/become-03.png",
          "key": "brand-become-03",
          "number": 80,
          "title": "Leverage value within your market",
        },
      ]
    }
    title="Why you should activate Data Labs for your Brand?"
  />
  <ForwardRef(LoadableComponent)
    defaultReason={4}
  />
  <Memo(Reviews) />
  <Memo(JoinUs)
    defaultReason={4}
    description="Activate Data Labs for your brand"
  />
</Fragment>
`;
